CMakeLists.txt

```diff
@@ -364,7 +364,9 @@ set(LIT_ARGS_DEFAULT "-sv")
 if (MSVC OR XCODE)
   set(LIT_ARGS_DEFAULT "${LIT_ARGS_DEFAULT} --no-progress-bar")
 endif()
-set(LLVM_LIT_ARGS "${LIT_ARGS_DEFAULT}" CACHE STRING "Default options for lit")
+
+# HLSL Change - disable short output mode for tests (will list all tests)
+set(LLVM_LIT_ARGS "-v --no-progress-bar" CACHE STRING "Options for lit")
```

Comment:
Please don't make this the default. The default output is _much_ nicer in the terminal if you have an interactive session.